examples/cis2-multi/src/lib.rs

```diff
@@ -548,7 +549,12 @@ pub struct ViewState {
 
 /// View function for testing. This reports on the entire state of the contract
 /// for testing purposes.
-#[receive(contract = "cis2_multi", name = "view", return_value = "ViewState")]
+#[receive(
+    contract = "cis2_multi",
+    name = "view",
+    return_value = "ViewState",
+    error = "ContractError"
+)]
```

Comment:
I think this is incorrect, ReceiveResult has `Reject` as the error type.

Only `ContractResult` has ContractError.